### ISSUE TITLE
Add term Palpebral oculogyric reflex

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10350]
**Preferred term label:**
Reduced palpebral oculogyric reflex

**Synonyms**
Reduced bell's phenomenon, poor Bell's phenomenon

**Definition (free text, please give PubMed ID)**
A protective reflex in which the globe is turned upwards and slightly outwards during the eyelid closure to avoid corneal exposur
PMID: 6732654

**Parent term (use hpo.jax.org/app)**
Abnormal reflex HP:0031826

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM218200
OMIM139393

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
A protective reflex in which the globe is turned upwards and slightly outwards during the eyelid closure to avoid corneal exposure -- this is a normal finding? What is the medical context and can we provide a precise definition?

**Comment by azankl** [comment]
@pnrobinson Lisanne has left and I am not sure what she means with that either. We decided to focus on adding only skeletal terms, so OK to drop this request.

**Comment by pnrobinson** [comment]
dropping!
